src/Webform.js

```diff
@@ -1259,6 +1259,10 @@ export default class Webform extends NestedDataComponent {
             errors = [errors];
         }
 
+        if (Array.isArray(this.errors)) {
+            errors = _.union(errors, this.errors);
+        }
```

Comment:
We're trying to use native Javascript methods as much as possible.  here that would look like:
`errors = [...errors, ... this.errors];`